From e12b785dfa78e6f76c78f1dc3c78c9bea1a5d3da Mon Sep 17 00:00:00 2001
From: Josue David Hernandez <josue.d.hernandez.gutierrez@intel.com>
Date: Tue, 3 Jul 2018 18:28:54 -0500
Subject: [PATCH] Add /var/cache to XDG_DATA_DIRS var

Signed-off-by: Josue David Hernandez <josue.d.hernandez.gutierrez@intel.com>
---
 env.d/flatpak.env.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/env.d/flatpak.env.in b/env.d/flatpak.env.in
index 478770b..60d8c93 100644
--- a/env.d/flatpak.env.in
+++ b/env.d/flatpak.env.in
@@ -1 +1 @@
-XDG_DATA_DIRS=$HOME/.local/share/flatpak/exports/share/:@localstatedir@/lib/flatpak/exports/share/:/usr/local/share/:/usr/share/
+XDG_DATA_DIRS=$HOME/.local/share/flatpak/exports/share/:@localstatedir@/lib/flatpak/exports/share/:/usr/local/share/:/var/cache/:/usr/share/
-- 
2.18.0

